### PR TITLE
Feature - remove inspector declaration for first submission

### PR DESCRIPTION
**Pull request by joefitter**
* only show the inspector declaration if the project has an existing task

### CODE REVIEW

**Comment by joefitter** [comment]
This will likely break most of the tests so marking as WIP until a test fix is ready to go

File: pages/project-version/update/submit/index.js [hunk 1]
```diff
@@ -40,7 +40,8 @@ module.exports = settings => {
   app.use(
     form({
       configure: (req, res, next) => {
-        const schema = getSchema(req.version.type, req.user.profile.asruUser);
+        const hasAuthority = get(req.project, 'openTasks[0].data.meta.authority') === 'Yes';
```
Comment (lennym) [inline pages/project-version/update/submit/index.js:43]:
```suggestion
        // if application has previously been approved then this is a resubmission and we can show the inspector ready question
        const hasAuthority = get(req.project, 'openTasks[0].data.meta.authority') === 'Yes';
```

Comment (lennym) [reply pages/project-version/update/submit/index.js:43]: Because it's a hack.

Comment (joefitter) [reply pages/project-version/update/submit/index.js:44]: heh

**Pull request merged**